### PR TITLE
Updated version of the tool

### PR DESCRIPTION
**Pull request by FedericoTartarini**
Main contributions:
- Fixed GitHub issues #16, #3
- Added heat loss vs air temperature chart
- Added Upload page
- Fixed the air velocity chart
- Updated help files
- Removed LEED (as suggested by Stefano)
- Improved chart graphics
- Added Boostrap to the webpage so now the website is responsive and mobile friendly.
- Updated the graphics (navigation bar, footnote, charts color) using same images and colour as the official CBE website.
- Added link to MRT tool developed by Tyler
- Updated Google Analytics code (now we can track traffic in all pages), created a Gmail account for the website and added contact email.
- Change list of Authors (as suggested by Stefano)
- Add the psychrometric variables to the RH-dry-bulb chart
- created a global.js which contains shared functions
- Added unittest module for Python functions
- Fixed help me
- Moved from Python 2 to 3

For more information please look at the list of commits below. In addition, please access the update version of this tool using this [link](https://shrouded-refuge-24438.herokuapp.com)

### CODE REVIEW

**Comment by chriswmackey** [comment]
@FedericoTartarini ,

I just wanted to note again that I plan to send a PR for the Python version that is compatible with both Python 2 and 3, does not require any dependencies, includes all of [the documentation that you see on the ladybug comfort repo](https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/pmv.py#L11-L61), and includes [all of our unittests](https://github.com/ladybug-tools/ladybug-comfort/blob/master/tests/pmv_test.py)

If you want to keep editing the python here, that is fine but this is really starting to feel like duplicated efforts and it might just be best for you to signal to me the best time for me to send my PR (when you are not editing the code base here).

**Comment by FedericoTartarini** [comment]
Hi Chris, Tyler has to approve the current Pull Request and then you can edit the Python code. Please keep in mind that the Python code is now used by Flask to calculate the comfort parameters (PMV, CE, PPD, etc.) in the Update tab. The Python function comfPMVElevatedAirspeed() now returns a dictionary and not a list.

**Comment by chriswmackey** [comment]
Thanks for clarifying @FedericoTartarini . I'll aim to not change the output structure (I'm also using a dictionary in my newer code) or the names of the functions in any edits I make here.  Also, once we have this merged, I'll take some time to try and understand how you are using the Python code on the back end with Flask.

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: gunicorn comfort:app
```
Comment (thoyt) [inline Procfile]: What's this file for?

File: README.md [hunk 2]
```diff
@@ -1,44 +1,54 @@
...
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-Once you have pip, start by installing virutalenv:
+## Prerequisites
 
-`$ sudo pip install virtualenv`
+Python 3 installed on your machine.
 
-Next, check out from the repo and create a virtualenv:
+If you’re using Python 2, see [Install virtualenv](https://flask.palletsprojects.com/en/1.1.x/installation/#install-install-virtualenv) first.
```
Comment (thoyt) [inline README.md]: Does Python2 still work?

File: comfort.py [hunk 3]
```diff
@@ -1,60 +1,77 @@
 import os
 import csv
+from io import TextIOWrapper, StringIO
```
Comment (thoyt) [inline comfort.py]: `StringIO` unused?

**Comment by FedericoTartarini** [comment]
Hi @thoyt unfortunately I do not think I will be able to deploy it myself. Ideally it would be great if we could have it up and running by Monday next week so Stefano can show it during the CBE symposium/meeting. I will try to address all your comments as soon as possible.

File: runtime.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+python-3.7.3
```
[the remark this answers is not in the document]
Comment (FedericoTartarini) [reply runtime.txt]: I created it to Upload the code on Heroku. I have removed it.

File: Procfile (hunk 1 above)
Comment (FedericoTartarini) [reply Procfile]: For Heroku

Comment (FedericoTartarini) [reply Procfile]: Removed the file

File: README.md (hunk 2 above)
Comment (FedericoTartarini) [reply README.md]: I have not tested that, I guess so but I will remove that comment.

File: comfort.py (hunk 3 above)
Comment (FedericoTartarini) [reply comfort.py]: Removed StringIO

**Comment by FedericoTartarini** [comment]
Heroku requires the Procfile and runtime.txt files and cannot run application without them. I have currently removed them from master branch as you requested but now Stefano cannot longer access the application on Heroku which is connected to the Master branch.

**Comment by thoyt** [comment]
Would it help to use a special branch for the heroku deployment?

**Comment by FedericoTartarini** [comment]
it would, but at the same time I would have to update the code in both branches (master and special). In addition we are planning to use Heroku only for the testing phase and for nothing else, hence, once we are done with the testing I am planning to take the Heroku app down.

Arens has messaged me since he cannot access the Heroku app anymore. How long will it take to have the new version of the CBE tool online? If it is going to take a week I will have to commit to the master branch the two files I have removed

**Comment by FedericoTartarini** [comment]
To solve the issue with Heroku, I have now stopped automatic deployments on Heroku.

**Comment by thoyt** [comment]
I'm going to merge this into a special branch for deployment. I can take care of the extraneous files and it shouldn't affect your heroku deployment.

**Pull request merged**